lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -621,6 +627,42 @@ packThunk' noTrail thunkDir = checkThunkDirectory thunkDir >> readThunk thunkDir
       liftIO $ createThunk thunkDir thunkPtr
       pure thunkPtr
 
+-- set thunk to a desired branch
+setThunk :: MonadObelisk m => String -> FilePath -> m ()
```

Comment:
Could this be a thin wrapper around `uriThunkPtr`?

Reply:
Ah! Thanks for pointing `uriThunkPtr` out.